<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseRecords</class>
 <widget class="QDialog" name="BrowseRecords">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browse Record</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Filter By Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="recordsWidjet"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="parmeterscontainer" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="Parmeters">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Output Frame Every </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="offset">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>200</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Second</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="gropbox">
              <property name="title">
               <string>Columns</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QScrollArea" name="scrollarea">
                   <property name="widgetResizable">
                    <bool>true</bool>
                   </property>
                   <widget class="QWidget" name="widget_2">
                    <property name="geometry">
                     <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>278</width>
                      <height>506</height>
                     </rect>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_3">
                     <item row="0" column="2">
                      <widget class="QListWidget" name="columnsBox"/>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="PathLayout">
       <item>
        <widget class="QLineEdit" name="Path">
         <property name="placeholderText">
          <string>Enter Path To Save Or Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Extract</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
